assets/js/state/sagas/hosts.test.js

```diff
@@ -84,15 +84,15 @@ describe('Hosts sagas', () => {
   it('should send host deregister request', async () => {
     const host = hostFactory.build();
 
+    const mockNavigate = (route) => route;
```

Comment:
As we have chosen to always provide a function `navigate()` to the action payload for the `deregisterHost()` Saga, now in the test we need to provide a mocked `navigate` function. Is this function fine?

Reply:
You could have just a () => {} but this is fine as well